source/iostreams.tex

```diff
@@ -11796,11 +11796,11 @@
     friend bool operator==(const path& lhs, const path& rhs) noexcept;
     friend strong_ordering operator<=>(const path& lhs, const path& rhs) noexcept;
 
-    friend path operator/ (const path& lhs, const path& rhs);
+    friend path operator/(const path& lhs, const path& rhs);
 
     // \ref{fs.path.native.obs}, native format observers
     const string_type& native() const noexcept;
-    const value_type*  c_str() const noexcept;
+    const value_type* c_str() const noexcept;
```

Comment:
I didn't notice that this was actually aligned. The parentheses are what stood out to me, not the name. I will keep only the other change, whose alignment became outdated with the addition of `operator<=>`.